dfreader_pandas.py

```diff
@@ -0,0 +1,215 @@
+"""
+Module to parse ArduPilot logs into pandas DataFrames with optional caching.
```

Comment:
should be in tools/ or examples/

Reply:
Yeah, it's out of place at top level for sure. I do want it to be importable though, like `from pymavlink.tools.dfreader_pandas import parse_log_to_df`. I'll figure out how to do that. I think that means I need to add tools to packages, which now that I say it, is probably better than installing all these tools as scripts. We should be calling them like `python -m pymavlink.tools.magfit`